### ISSUE TITLE
Global protocol version & changelog

### ISSUE THREAD

**Issue by psiemens** [onflow/flow#405]
## Context

In order to communicate compatibility across different components of the Flow ecosystem, we want to define a **global protocol version** that defines the current version of the Flow protocol.

This version will not be tied to any specific codebase; instead, it is incremented when the semantics of the protocol change.

For example, we may have the following:

```
Flow (protocol): v1.0.4

flow-go v0.15.2 implements Flow v1.0.4
flow-emulator v0.8.1 implements Flow v1.0.4

flow-js-sdk v3.2.0 is compatible with Flow v1.0.4
flow-go-sdk v0.4.0 is compatible with Flow v1.0.4
```

Each individual library or tool is free to have its own versioning system -- the protocol version is what determines compatibility between all of them.

## Definition of Done

- [ ] Add Flow protocol changelog to docs site
- [ ] On changelog page, include description of the _Flow protocol version_
- [ ] After determining the first protocol version, _create a release on this repository_ (`onflow/flow`)

## PR Review 

Who is your code partner: @psiemens

**Comment by 10thfloor** [comment]
@psiemens Can we talk about this one again soon?

**Comment by psiemens** [comment]
Yep let's talk about it again tomorrow, timing will be perfect for Wednesday's spork. 

TL;DR is I think there's a light version of this we can get out for Wednesday that provides clarity about the spork changes and the relevant SDK-level changes that go with it.

**Comment by psiemens** [comment]
Releases compatible with Mainnet 6:

Flow Go: https://github.com/onflow/flow-go/releases/tag/v0.14.8
Emulator: https://github.com/onflow/flow-emulator/releases/tag/v0.16.0
Flow Go SDK: https://github.com/onflow/flow-go-sdk/releases/tag/v0.16.1

Mainnet 6 is compatible with ___ version of JS libraries
